wcfsetup/install/files/lib/system/email/transport/SmtpEmailTransport.class.php

```diff
@@ -468,6 +468,8 @@ public function deliver(Email $email, Mailbox $envelopeFrom, Mailbox $envelopeTo
             return $item;
         }, \explode("\r\n", $email->getEmail()))) . "\r\n");
         $this->write(".");
-        $this->read([250]);
+        [$code, $message] = $this->read([250]);
```

Comment:
```suggestion
        [, $message] = $this->read([250]);
```
Rationale: `$code` was never used and this makes it clear that this is intentional.